artman/tasks/task_base.py

```diff
@@ -52,7 +52,7 @@ def exec_command(self, args):
 
         TODO(ethanbao): Use subprocess.Popen which is recommended."""
         try:
-            self.log(' '.join(args), level=logging.DEBUG)
+            self.log(' '.join(args), level=logging.INFO)
```

Comment:
Did you mean to keep this change in, or was this just for debugging?

Reply:
I added initially because of debugging, but then concluded that it was nice to know what was actually going on when watching artman execute.

Reply:
INFO is the usual default. I _think_ artman has a flag for getting DEBUG output.

Reply:
Well artman was a little inconsistent before - it would print lines for *some* tasks that were going to execute, but not others, giving the wrong impression of how long particular steps would take. Do we want to remove those other lines that are only partially informative?

Reply:
Up to you. :-)

Reply:
I decided to make it quieter. Adding `-v` on the command line is very easy.